index.html

```diff
@@ -479,3 +515,13 @@ <h2 class="alignCenter">Hi!</h2>
     <script defer src="thirdparty/jquery/jquery-2.1.1.min.js"></script>
   </body>
 </html>
+<html><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8"><meta name="Robots" content="NOINDEX " /></head><body></body>
```

Comment:
Second `<html>` tag is invalid! Please move this into existing `<head>` tag (without `<html`> of course).